### PR TITLE
 Remove some more unncessary/doubled map check

### PR DESCRIPTION
**Pull request by KennyTV**
Cleaned up the easy double checks, tested it on a lobbyserver (1.8 with 1.8-1.13 support) with about 50 players and looked over it a second and third time to make sure I didn't negate any if-clauses

### CODE REVIEW

**Comment by MylesIsCool** [comment]
Looks good to me, need someone to double check over just in case I missed something :P

File: bungee/src/main/java/us/myles/ViaVersion/bungee/service/ProtocolDetectorService.java [hunk 1]
```diff
@@ -27,16 +27,19 @@ public ProtocolDetectorService(BungeePlugin plugin) {
...
+        Integer detectedProtocol = detectedProtocolIds.get(serverName);
+        if (detectedProtocol != null) {
+            return detectedProtocol;
         }
         // Step 3. Use Default
-        if (servers.containsKey("default")) {
-            return servers.get("default");
+        Integer defaultProtocol = servers.get("default");
+        if (defaultProtocol != null) {
+            return detectedProtocol;
```
Comment (Plancke) [inline bungee/src/main/java/us/myles/ViaVersion/bungee/service/ProtocolDetectorService.java]: This should probably return the default protocol?

File: common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/storage/EntityTracker.java [hunk 2]
```diff
@@ -59,13 +59,13 @@ public EntityTracker(UserConnection user) {
...
 
     public UUID getEntityUUID(int id) {
-        if (uuidMap.containsKey(id)) {
-            return uuidMap.get(id);
-        } else {
-            UUID uuid = UUID.randomUUID();
+        UUID uuid = uuidMap.get(id);
+        if (uuid == null) {
+            uuid = UUID.randomUUID();
             uuidMap.put(id, uuid);
```
Comment (Plancke) [inline common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/storage/EntityTracker.java]: ComputeIfAbsent would probably be faster here.

File: bungee/src/main/java/us/myles/ViaVersion/bungee/service/ProtocolDetectorService.java (hunk 1 above)
Comment (KennyTV) [reply bungee/src/main/java/us/myles/ViaVersion/bungee/service/ProtocolDetectorService.java]: My bad, would have completely missed it.. Thanks!

File: common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/storage/EntityTracker.java (hunk 2 above)
Comment (KennyTV) [reply common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/storage/EntityTracker.java]: The project is set to Java 7 afaik

Comment (Plancke) [reply common/src/main/java/us/myles/ViaVersion/protocols/protocol1_9to1_8/storage/EntityTracker.java]: Oh yeah just checked and it is, that's unfortunate ^^

**Comment by MylesIsCool** [comment]
Thanks @KennyTV @Plancke much appreciated <33

**Pull request merged**